pkg/network/namescheme/networknamescheme.go

```diff
@@ -19,4 +19,35 @@
 
 package namescheme
 
+import (
+	"fmt"
+
+	v1 "kubevirt.io/api/core/v1"
+
+	"kubevirt.io/kubevirt/pkg/network/vmispec"
+)
+
 const PrimaryPodInterfaceName = "eth0"
+
+func CreateNetworkNameScheme(vmi *v1.VirtualMachineInstance) map[string]string {
```

Comment:
Please consider changingg the func signature to receive []v1.Network, its seems that its the only thing that being used.

nit: I would call this func `Generate..` instead of `Create..` because it could emphasize that the network-name-scheme is not created out of anything.
There is an algorithm that processes the data we have (vmi.spec.networks) and produces something out of it. 
Not a must.

Reply:
>Please consider changing the func signature to receive []v1.Network, it seems that it's the only thing that is used. 

Done

>nit: I would call this func Generate..

I think create and generate have the same meaning in this scope.